include/deal.II/grid/grid_tools.h

```diff
@@ -468,20 +468,27 @@ namespace GridTools
   /*@{*/
 
   /**
-   * Find and return the number of the used vertex in a given mesh that is
-   * located closest to a given point.
+   * Find and return the number of the used vertex (or marked vertex) in a
+   * given mesh that is located closest to a given point.
    *
    * @param mesh A variable of a type that satisfies the requirements of the
    * @ref ConceptMeshType "MeshType concept".
    * @param p The point for which we want to find the closest vertex.
+   * @param[Optional] marked_vertices An array of bools indicating whether an
+   * entry in the vertex array should be considered (and the others must
+   * be ignored) as the potentially closest vertex.
    * @return The index of the closest vertex found.
    *
+   * @note On receiving a non-default @p marked_vertices, the function will
```

Comment:
non-default -> non-empty, but i think this note should be combined with the description of `marked_vertices`.